doc/Programming-guide/Key-platform-features/Common-Type-Library/index.md

```diff
@@ -7,6 +7,8 @@ sort_idx: 20
 {% assign root_url = page.url | split: '/'%}
 {% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
 
+**Table of Contents**
```

Comment:
Please revert this. According to the doc style guide you don't need a header for a TOC.